internal/commands/function/command.go

```diff
@@ -0,0 +1,138 @@
+package function
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/briandowns/spinner"
+
+	"github.com/spf13/pflag"
+)
+
+// set of supported `push` command strings
```

Comment:
[mega-nit] small copy-paste things here and on a few other lines in the file. Should be `function` instead of `push`?

Reply:
Good eye